docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -0,0 +1,561 @@
+        "lpar1": {
+            "properties": {
+                "host": "example1.com"
+            },
+            "profiles": {
+                "zosmf": {
+                    "type": "zosmf",
+                    "properties": {
+                        "port": 443
+                    }
+                }
+            }
+        }
+    },
+    "defaults": {
+        "zosmf": "lpar1.zosmf"
+    },
+    "plugins": [],
+    "secure": [
+        "profiles.lpar1.properties.user",
+        "profiles.lpar1.properties.password"
+    ]
+}
+```
+
+From here, you can edit the details as needed and save the file. For example, you might change the host or port fields if those values changed.
+
+To add a new service, for example add a new instance of z/OSMF that runs on a different mainframe LPAR, you can build on the existing array as follows:
+
+```json
+{
+    "$schema": "./zowe.schema.json",
+    "profiles": {
+        "lpar1": {
+            "properties": {
+                "host": "example1.com"
+            },
+            "profiles": {
+                "zosmf": {
+                    "type": "zosmf",
+                    "properties": {
+                        "port": 443
+                    }
+                }
+            }
+        },
+        "lpar2": {
+            "properties": {
+                "host": "example2.com"
+            },
+            "profiles": {
+                "zosmf": {
+                    "type": "zosmf",
+                    "properties": {
+                        "port": 1443
+                    }
+                }
+            }
+        }
+    },
+    "defaults": {
+        // Change to lpar2.zosmf if you wish to change default profile
+        "zosmf": "lpar1.zosmf"
+    },
+    "plugins": [],
+    "secure": [
+        "profiles.lpar1.properties.user",
+        "profiles.lpar1.properties.password",
+        // See below about tips for using base array to avoid duplicating these
+        "profiles.lpar2.properties.user",
+        "profiles.lpar2.properties.password"
+    ]
+}
+```
+
+You can continue to add more LPARs, and more services within each LPAR. After you make changes, save the file and issue a Zowe CLI command to the service to verify connection.
+## Managing credential security
+
+When you first run the `zowe config init --global` command, the `profiles.base.properties.user` and `profiles.base.properties.password` fields are defined to the "secure" array in your configuration file. This ensures that username and password are stored securely on your computer.
+
+Issue the `zowe config secure` command to re-prompt for all secure fields when you want to update them (for example, when you want to change your username and password).
+
+To secure a specific field, use the command `zowe config set secure --<option-name>`. For example, you can issue `zowe config set secure --host`. If you issue the command for an option that is already secure, the CLI re-prompts you to enter a new option value.
```

Comment:
The command for securing a specific field is `zowe config set --secure <property-path>`.

An example of this command would be `zowe config set --secure profiles.base.properties.password`.